core/commands/repo.go

```diff
@@ -245,25 +243,25 @@ daemons are running.
 
 		err = os.Remove(repoLockFile)
 		if err != nil && !os.IsNotExist(err) {
-			res.SetError(err, cmdkit.ErrNormal)
-			return
+			return err
 		}
 		err = os.Remove(dsLockFile)
 		if err != nil && !os.IsNotExist(err) {
-			res.SetError(err, cmdkit.ErrNormal)
-			return
+			return err
 		}
 		err = os.Remove(apiFile)
 		if err != nil && !os.IsNotExist(err) {
-			res.SetError(err, cmdkit.ErrNormal)
-			return
+			return err
 		}
 
-		res.SetOutput(&MessageOutput{"Lockfiles have been removed.\n"})
+		return cmds.EmitOnce(res, &MessageOutput{"Lockfiles have been removed."})
```

Comment:
Although minor, removing the newline is an API change.

@Stebalien is this okay?

Reply:
We should *probably* keep the newline.

Reply:
Sorry, this is my mistake. API should remain stable. Thx for helping me find this.